src/cryptography/hazmat/primitives/serialization/pkcs12.py

```diff
@@ -4,6 +4,17 @@
 
 from __future__ import absolute_import, division, print_function
 
+from cryptography.hazmat.backends import default_backend
+
 
 def load_key_and_certificates(data, password, backend):
     return backend.load_key_and_certificates_from_pkcs12(data, password)
+
+
+def serialize_key_and_certificates(name, key, cert, cas, key_encryption):
```

Comment:
It's not really `key_encryption`, it's just `encrytion_params`, right?

Reply:
Agree